Sources/Networking/HTTPResponse.swift

```diff
@@ -30,3 +30,70 @@ extension HTTPResponse: CustomStringConvertible {
     }
 
 }
+
+// MARK: -
+
+/// The response content of a failed request.
+struct ErrorResponse {
```

Comment:
nice!!! this will also help us get rid of some hacky stuff we have like the `finishableKey` and `RCSuccessfullySyncedKey`, which were just awkward ways to communicate information through the same existing dictionary back to other layers

Reply:
👍🏻